nimx/drag_and_drop.nim

```diff
@@ -0,0 +1,68 @@
+import view
+import types
+import class_registry
+import variant
+
+import nimx.image
+import nimx.pasteboard.pasteboard
+
+const DragPboardKindDefault* = "nimx.dragged.default"
+
+type BaseDragAndDrop* = ref object of DragAndDrop
```

Comment:
There's no need for this class. The delegates should subclass directly from `DragDestinationDelegate`